apps/authentication/confirm/mfa.py

```diff
@@ -1,11 +1,12 @@
 from users.models import User
 
 from .base import BaseConfirm
+from ..const import ConfirmType
 
 
 class ConfirmMFA(BaseConfirm):
-    name = 'mfa'
-    display_name = 'MFA'
+    name = ConfirmType.MFA.value
+    display_name = ConfirmType.MFA.name
 
     def check(self):
         return self.user.active_mfa_backends and self.user.mfa_enabled
```

Comment:
There are no significant differences between the provided code snippets. Both codes seem to be correctly implemented for configuring MFA using the `users.models.User` model and specifying the backend as either Google Authenticator (Base) or Microsoft authy(MFA). No optimizations suggested here since there are minimal changes in the implementation.